src/app/organization/organization.component.html

```diff
@@ -1,45 +1,77 @@
-<app-navbar></app-navbar>
-
-<div class="container-fluid">
-  <div class="row details-wrapper">
-    <div class="col">
-      <h1 class="org-title"> @JBossOutreach </h1>
-      <h4> Mentor @ JBoss </h4>
-      <h4> Mumbai, IN </h4>
-    </div>
-    <div class="col rtl" >
-      <h1 class="org-title"> Certificates: {{ this.rowData.length }} </h1>
-      <h4> Download </h4>
-    </div>
-    <div class="container-fluid">
-      <div class="row action-wrapper">
-        <div class="col-2"></div>
-        <button class="col-lg-3"> IMPORT </button>
-        <div class="col-2"></div>
-        <button type="button" routerLink="/issue/single" class="col-lg-3 blue-btn"> GENERATE </button>
-        <div class="col-2"></div>
-      </div>
-    </div>
+<div class="nav-bar">
+    <span class="logo">
+        <a href="#">Certificate Generator</a>
+    </span>
+    <span class="links">
+        <a href="home">Home</a>
+        <a href="signup">SignUp</a>
```

Comment:
Does this actually works?

Reply:
the nav? no

Reply:
Add proper routers then if you want us to consider this for merge, task wise all ok